tidy3d/components/medium.py

```diff
@@ -2840,6 +2870,53 @@ def _all_larger(coeff_a, coeff_b) -> bool:
             return np.all(coeff_a.values > coeff_b.values)
         return coeff_a > coeff_b
 
+    @classmethod
+    def from_nk(cls, n: float, k: float, freq: float, **kwargs):
+        """Convert ``n`` and ``k`` values at frequency ``freq`` to :class:`Lorentz`.
```

Comment:
can you add some description of how this is being done? it's hard to tell from the path? for example that there is a single pole and how its computed?

Reply:
It's a single-pole Lorentz model. How it is computed is obvious from the code? I have a PDF note on this, but I think the code is simple enough?

Reply:
yea it's obvious from the code, I just mean that someone looking at the API reference might just be curious, so a little info could be useful (even if just mentioning that it is a **single** pole Lorentz model).

Reply:
i see, that has been added in the revision.